packages/elements/__tests__/setup.ts

```diff
@@ -41,6 +42,45 @@ Object.defineProperty(window, "matchMedia", {
   })),
 });
 
+// Patch createElement to handle SVG elements properly in jsdom
+const SVG_TAGS = new Set([
+  "svg",
+  "path",
+  "circle",
+  "g",
+  "rect",
+  "line",
+  "polyline",
+  "polygon",
+  "ellipse",
+  "text",
+  "tspan",
+  "defs",
+  "clipPath",
+]);
+
+const originalCreateElement = document.createElement.bind(document);
```

Comment:
Not the most exciting fix for the SVG issue, but for now it works. I do think there might be merit to migrate to Vitest Browser Mode eventually: https://github.com/vercel/ai-elements/issues/169